html/index.php

```diff
@@ -20,6 +20,11 @@ function newtweet($tweet_textarea)
     // 汎用ログインチェック処理をルータに作る。早期リターンで
     createTweet($tweet_textarea, $_SESSION['user_id']);
 }
+
+function newReplyTweet($tweet_textarea,$reply_post_id)
```

Comment:
引数の間にスペースをいれてください
（他の部分と統一させるため）

実際の現場でもスペースいれることが多いですが、最低限1つのプロジェクト内で体裁が整っていれば大丈夫です
今回作ってもらっているコードの中で統一できていないので指摘しています

```suggestion
function newReplyTweet($tweet_textarea, $reply_post_id)
```